### ISSUE TITLE
Please enable 2FA in user's mailcow UI as it is available for sys admin and domain admins

### ISSUE THREAD

**Issue by DepriMarvin** [mailcow/mailcow-dockerized#4494]
## Summary

Enable 2FA (e.g. TOTP) for the user's mailcow UI.
The code is already there for admin access.

## Motivation

In my oppinion it does not make any sense to have 2FA in SOGo, but not in mailcow UI.
From mailcow UI the user can login to SOGo without the 2nd factor, event if it is set in SOGo.

I think, having a 2FA for the mailcow UI is a must have for users.
Having a FIDO2 device is also not the solution, because:
1. Not everyone has got a FIDO2 device. TOTP with authenticator apps on the other hand comes with no additional costs for users.
2. FIDO2 is not exclusive, if set. You still can login with user & password only.

If an attacker gets access to the user's mailcow UI, he can compromise the whole user and also the whole server, if it is afterwards used as a SPAM source.

The "Login to webmail" on the user's mailcow UI cannot be disabled with any configuration setting.

From the mailcow UI the user's password and app passwords can be set and FIDO2s can be added and deleted.
And even other information can be gathered from there for example temporary aliases and other mailboxes via Sync jobs.

With bypassing the 2FA of SOGo over the mailcow UI attackers can also get access to all the DAV data (calender and even worse contacts).

**Comment by MrFogHunter** [comment]
Hello I totally agree with the creator of @DepriMarvin 
I agree on the danger but I think it's in progress after what I read 
I have to admit that it's weird to come across this kind of bug

**Comment by snevas** [comment]
Your feature request is much needed here as well, but for a different reason: I use the mailcow UI flow to securely login. Right now I have to abuse the domain / admin user to get the possibly to set a TFA method / or use FIDO2 on the mailbox-level (with the downside you descibed above of still having password login without TFA enabled).
In my case I only use app passwords for my clients and the single-sign-on to SOGo-button for webmail. All other direct, external access is disabled.

> Not everyone has got a FIDO2 device. TOTP with authenticator apps on the other hand comes with no additional costs for users.

Also passwordless or 2nd factor WebAuthn does not require you to buy a device. You can use a Android phone or even Windows Hello (fingerprint / pin) as a device as well.
But sadly mailcow implemented it in a way that Android does not work, but windows Hello does. You can test the Android flow from a Chrome browser here: https://webauthn.io/

TLDR; please enable TFA on mailbox-users as well in the mailcow UI

**Comment by MAGICCC** [comment]
Since https://github.com/mailcow/mailcow-dockerized/releases/tag/2022-07 is released it you can use it TFA as a normal user
